### DISCUSSION TITLE
What happens if one of my cache disks broken?

### DISCUSSION THREAD

**Discussion by zkryakgul** [minio/minio#14365] [Q&A] [answered]
Hi guys,

I have a minio deployment which contains 2 serverpool with 2 servers. All of my servers have an directory `/data/miniocache` which is consist of 6 x 960G SSD disks configured as **RAID 0**. Here is my questions: 

1. What happens if one of my disks broken? 
2. Is all of my cache will dissepear or i only lost cache data in broken drive?
3. Is it enough to just replace the corrupted disk?

Thanks!

**Comment by klauspost** [comment, marked as answer]
@zkryakgul Isn't that how RAID 0 works?

Not sure what you are asking. If your cache disk disappears your cache is gone.

Replacing it should bring back functionality.
